### DISCUSSION TITLE
How to switch left & right mouse clicks events?

### DISCUSSION THREAD

**Discussion by Nokthed** [google/model-viewer#4617] [Q&A] [answered]
Oy,
I've been told navigating my map with model-viewer was 'annoying' when using a trackpad. The suggestion I've been given was to switch the mouse clicks events; left mouse to navigate the map and right mouse for pan.

Any idea how this could be done?
Thanks in advance.

**Comment by Nokthed** [comment]
bump

**Reply by elalish** [reply]
You can leave `camera-controls` off and then use your own pointer event listeners to change `cameraTarget` etc. If some of our interactions are still useful you can also consider something like `disable-pan` or `disable-tap`: https://modelviewer.dev/docs/index.html#entrydocs-stagingandcameras-attributes-disablePan

**Reply by Nokthed** [reply]
Thanks for your response, I appreciate it.
Would it be possible to be led to some examples or something similar ? I'm just an amateur with javascript so I'm unsure how I can achieve this. :/

**Reply by elalish** [reply, marked as answer]
If you're an amateur with JS, then start by playing with the attributes from our API. If that isn't enough, then find a JS class and learn about handling pointer events - it'll take some work to do anything useful at that lower level.
